dags/main_summary.py

```diff
@@ -62,7 +62,17 @@
                       output_visibility="private",
                       dag=dag)
 
+t6 = EMRSparkOperator(task_id="main_events",
+                      job_name="Main Events View",
+                      execution_timeout=timedelta(hours=4),
+                      release_label="emr-5.2.1",
+                      instance_count=1,
```

Comment:
Yes -- currently this job is taking about 10 minutes per day. I'm sure that'll go up as more people adopt events but for now this is fine.